src/chapters/css/exercises.rst

```diff
@@ -0,0 +1,13 @@
+Exercises: CSS
+==============
+
+We have built a website for you to test your CSS knowledge on. Check out the `repl.it <https://repl.it/@launchcode/CSSExercises/>`_!
+
+For the exercises, add the following style rules to ``style.css``:
```

Comment:
There are multiple tasks for styling elements (``p``, ``h1``, etc.) and one for styling an id, but there are no direct examples for styling a *class*. Students could create a ``centered`` class, for example, and apply that inside the tags, or they could add ``text-align: center;`` to each element in the CSS.

I would vote for adding another ``id`` example and a couple of ``class`` examples in the exercises, especially if they can be structured to reinforce the priority for element/class/id styling.

Reply:
I think I got some good new exercises! Thanks!